tests/unit/records/test_schemas.py

```diff
@@ -72,3 +72,37 @@ def test_contributors(app, db, minimal_record):
         {'name': 'test', 'affiliation': 'test', 'type': 'Invalid'}
     ]
     pytest.raises(ValidationError, Record.create, minimal_record)
+# validation of full_record fixture
+
+
+def test_full_json(app, db, full_record):
+    """Test full json."""
+    Record.create(full_record)
+
+
+def full_test_recid(app, db, full_record):
+    """Test recid property."""
+    full_record['recid'] = '12345'
+    pytest.raises(ValidationError, Record.create, full_record)
+
+
+def full_test_resource_type(app, db, full_record):
+    """Test resource type."""
+    full_record['resource_type'] = 'publication'
+    pytest.raises(ValidationError, Record.create, full_record)
```

Comment:
I know this was not in other tests, but let's introduce the following convention for this type of testing:

```
exc = pytest.raises(ValidationError, Record.create, full_record)
assert exc.value.message == "'publication' is not of type 'object'"
```
i.e. every time you check for validation error raising, additionally assert that the cause is as expected.